packit/constants.py

```diff
@@ -38,3 +38,25 @@
 GH2FED_RELEASE_TOPIC = "org.fedoraproject.prod.github.release"
 
 DEFAULT_BODHI_NOTE = "New upstream release: {version}"
+
+PACKIT_CONFIG_TEMPLATE = """# This configuration file is generated. See the documentation for more information:
```

Comment:
I'd probably remove the part about the config being generated: that usually means it's generated all the time, which is not the case here.